packages/video_player/video_player/lib/video_player.dart

```diff
@@ -914,11 +914,12 @@ class ClosedCaption extends StatelessWidget {
   /// Creates a a new closed caption, designed to be used with
   /// [VideoPlayerValue.caption].
   ///
-  /// If [text] is null, nothing will be displayed.
+  /// If [text] is null or empty, nothing will be displayed.
```

Comment:
Ideally `text` should be non-nullable with a default to `''` so that we don't have two ways of expressing the same thing, but unfortunately that would be a breaking change at this point.